topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,130 @@
+
+> ### {% icon hands_on %} Hands-on: Annotate genome
+>
+> 1. **Prokka** {% icon tool %} with the following parameters (leave everything else unchanged)
+>    - contigs to annotate: `contigs.fasta`
+>    - Locus tag prefix (--locustag): P
+>    - Force GenBank/ENA/DDJB compliance (--compliant): No
+>    - Sequencing Centre ID (--centre): V
+>    - Genus Name: Staphylococcus  
+>    - Species Name: aureus  
+>    - Use genus-specific BLAST database: No  
+>    - Your tool interface should look like this:
+>    - ![prokka interface](images/interface.png)
+>    - Click Execute
+> <br><br>
+{: .hands_on}
+
+## Examine the output
+
+Once Prokka has finished, examine each of its output files.
+
+ - The GFF and GBK files contain all of the information about the features annotated (in different formats.)
+ - The .txt file contains a summary of the number of features annotated.
+ - The .faa file contains the protein sequences of the genes annotated.
+ - The .ffn file contains the nucleotide sequences of the genes annotated.
+ <br><br>
+
+
+## View annotated features in JBrowse
+
+Now that we have annotated the draft genome sequence, we would like to view the sequence in the JBrowse genome viewer. First, we have to make a JBrowse file. Then, we can view it within Galaxy.
+
+> ### {% icon hands_on %} Hands-on: Create the JBrowse file
+>
+> 1. Search for **JBrowse** {% icon tool %} and run it with the following parameters
+>    - "Reference genome to display" to `Use a genome from history`
+>    - "Select the reference genome" to `Prokka on data XX.fna`.
+>       
+>       This sequence will be the reference against which annotations are displayed
+>
+>    - "Produce Standalone Instance" to `Yes`
+>    - "Genetic Code" to `11: The Bacterial, Archaeal and Plant Plastid Code`
+>    - "JBrowse-in-Galaxy Action" to `New JBrowse Instance`
+>    - "Track Group"
+>    - We will now set up one track - each track is a dataset displayed underneath the reference sequence (which is displayed as nucleotides in FASTA format).
+>    - We will choose to display the annotations (the Prokka.gff file).
+>
+>       - **Track 1 - sequence reads**: Click on `Insert Track Group` and fill it with
+>           - "Track Cateogry" to `gene annotations`
+>           - Click on `Insert Annotation Track` and fill it with
+>               - "Track Type" to `GFF/GFF3/BED/GBK Features`
+>               - "GFF/GFF3/BED Track Data" to `Prokka on data XX:gff`
+>               - "Track Visibility" to `On for new users`
+>               - "JBrowse Track Type [Advanced]" to `Canvas Features`
+>               - Click on "JBrowse Styling Options [Advanced]"
+>               - "JBrowse style.label" to `product,name,id`
+>               - "Track Visibility" to `On for new users`
+>               - Click Execute
+>
+>  - A new file will be created in your history, this contains the JBrowse interactive visualisation. We will now > view its contents and play with it
+> <br><br>
+> {: .hands_on}
```

Comment:
remove `>` at beginning here

Reply:
You can also probably merge the two hands-on together as https://github.com/galaxyproject/training-material/pull/667/files#diff-85eb4ab60195721a80f9728d9a3ac220R182

Reply:
yes, merged